src/pages/en/guides/markdown-content.md

```diff
@@ -312,6 +312,29 @@ import {Content as PromoBanner} from '../components/promoBanner.md';
 <PromoBanner />
 ```
 
+When used with `Astro.glob()`, you can simply pass the `<Content/>` component from the _Exported Properties_ of each returned markdown file. This will then render out the Markdown file and its contents in its entirety into your template. 
```

Comment:
Tried to rework this a bit given the code example is quite specific. I also removed “simply” because that can be quite subjective and potentially frustrating for someone if they’re finding the process anything but simple 😅 

```suggestion
When using `getStaticPaths` and `Astro.glob()` to generate pages from Markdown files, you can pass the `<Content/>` component through the page’s `props`. You can then retrieve the component from `Astro.props` and render it in your template. 
```